crates/rapid-web/src/server.rs

```diff
@@ -230,16 +230,11 @@ pub fn generate_typescript_types(bindings_out_dir: PathBuf, routes_dir: String,
 		routes_directory.clone()
 	};
 
-	// Show a loading spinner as needed
-	let loading = Spinach::new(format!("{} Generating types...", rapid_logo()));
```

Comment:
I would prefer if we leave the spinner.

Reply:
I've reintroduced the spinner along with the rapid_logo and made it so when its done, it just ends up like so
<img width="548" alt="image" src="https://github.com/Cincinnati-Ventures/rapid/assets/46626618/122b25c8-3dfe-4ac8-b3bb-74a96ecf4c0e">